<?xml version='1.0' encoding='UTF-8'?>
<ui version="4.0">
 <class>VCLFilterScreen</class>
 <widget class="QWidget" name="VCLFilterScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>132</width>
    <height>136</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>132</width>
    <height>136</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="mainFrame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QEDescriptionLabel" name="descriptionLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>13</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QEPushButton" name="insertButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Insert</string>
        </property>
        <property name="clickText">
         <string>2</string>
        </property>
        <property name="defaultStyle" stdset="0">
         <string>QEPushButton{ background-color: #a0ffa0; color: #000000; }</string>
        </property>
        <property name="displayAlarmStateOption" stdset="0">
         <enum>QE::Never</enum>
        </property>
        <property name="format" stdset="0">
         <enum>QE::Integer</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QEPushButton" name="withdrawButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Withdraw</string>
        </property>
        <property name="clickText">
         <string>1</string>
        </property>
        <property name="defaultStyle" stdset="0">
         <string>QEPushButton{ background-color: #ffa0a0; color: #000000; }</string>
        </property>
        <property name="displayAlarmStateOption" stdset="0">
         <enum>QE::Never</enum>
        </property>
        <property name="format" stdset="0">
         <enum>QE::Integer</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QELabel" name="statusLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>17</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>17</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QEPvFrame" name="statusImage">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="variableAsToolTip">
         <bool>true</bool>
        </property>
        <property name="displayAlarmStateOption" stdset="0">
         <enum>QE::Never</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QEDescriptionLabel</class>
   <extends>QELabel</extends>
   <header>QEDescriptionLabel.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QEPvFrame</class>
   <extends>QEFrame</extends>
   <header>QEPvFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QEFrame</class>
   <extends>QFrame</extends>
   <header>QEFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vcl_filter_screen.qrc"></include>
 </resources>
 <connections></connections>
</ui>
